console-src/src/queries/index.tsx

```diff
@@ -0,0 +1,19 @@
+export const fetchClientsQuery = `
+  {
+    viewer {
+      clients {
+        id
+        redirectUri
+        name
+        grantTypes
+        scopes
+      }
+    }
+  }
+`;
+
+export const createClientMutation = ``;
```

Comment:
This is of course temporary, I will fill in these queries as I implement the corresponding UI elements